website/docusaurus.config.js

```diff
@@ -21,12 +21,26 @@ const assert = require('assert');
 
 const darkCodeTheme = require('prism-react-renderer/themes/dracula');
 const lightCodeTheme = require('prism-react-renderer/themes/github');
+const semver = require('semver');
```

Comment:
SemVer does not exist on the `website` package, please include it in its `package.json`